docs/reference/snapshot-restore/snapshot-vers-compat.asciidoc

```diff
@@ -0,0 +1,30 @@
+
+[cols="^,^,^,^,^"]
+|====
+| 4+^h| Cluster version
+h| Snapshot version         | 5.0–5.6    | 6.0–6.8    | 7.1–{prev-major-last} | 8.0–{minor-version}
```

Comment:
> 8.0–{minor-version}

I'll amend this to just **8.0** in the 8.0 backport. I'll drop this column entirely in the 7.x backports.